pages/users/[id].tsx

```diff
@@ -9,7 +9,7 @@ import Footer from 'components/Footer'
 import Navbar from 'components/Navbar'
 import Loader from 'components/Loader'
 import { Container as OffsetBorderContainer } from 'components/OffsetBorder'
-import EventsPaginationButton from 'components/user/EventsPaginationButton'
+import EventsPaginationButton from 'components/PaginationButton'
```

Comment:
This could be renamed to `PaginationButton` like on the leaderboard page, to match up the names.